ceph-releases/jewel/ubuntu/14.04/daemon/entrypoint.sh

```diff
@@ -376,7 +380,7 @@ EOF
     chmod +x /etc/service/${CLUSTER}-${OSD_ID}/run
   done
 
-exec /sbin/my_init
+exec /usr/bin/runsvdir -P /etc/service
```

Comment:
Why not changing the path inside the `/sbin/my_init`?
I used to have that issue and at some point it was related to the path of the binaries.